modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements_controller.rb

```diff
@@ -45,16 +46,16 @@ def schema
   def validate_json_schema
     validate_json_schema_for_headers
     validate_json_schema_for_body
-  rescue JsonSchema::JsonApiMissingAttribute => e
-    render json: e.to_json_api, status: e.code
+  rescue SCHEMA_ERROR_TYPE => e
+    render json: { errors: e.errors }, status: 422
   end
 
   def validate_json_schema_for_headers
-    AppealsApi::FormSchemas.new.validate!("#{self.class::FORM_NUMBER}_HEADERS", headers)
+    AppealsApi::FormSchemas.new(SCHEMA_ERROR_TYPE).validate!("#{FORM_NUMBER}_HEADERS", headers)
```

Comment:
Something about the way this is reading is yelling at me. Why are we validating against an error type?